p/p7zip/stone.yaml

```diff
@@ -0,0 +1,24 @@
+name: 7zip
+version: 24.09
+release: 1
+summary: 7-Zip command-line version for Linux
+license: LGPL-2.1-or-later
```

Comment:
It's also licensed under bsd-3. maybe that part isn't built by default?

Reply:
added it anyway